### ISSUE TITLE
Path Resolution Issue (Module not found)

### ISSUE THREAD

**Issue by Enteleform** [morulus/complex-loader#1]
### System

```
Windows:         10
Node:            11.5.0
Webpack:         4.29.6
complex-loader:  0.1.5
```

&nbsp;  
### Issue

```
ERROR in ./src/Pages/SomePage/Layout.pug
Module not found: Error: Can't resolve 'C:ProjectsSomeProjectsrcPagesSomePageLayout.pug' in 'C:\Projects\SomeProject\src\Pages\SomePage'
 @ ./src/Pages/SomePage/Layout.pug 1:49-428
 @ multi (webpack)-inject-plugin/dist/webpack-inject-plugin.loader?id=webpack-inject-module-98bf ./src/Pages/SomePage/Layout.pug
```

&nbsp;  
It seems like the issue may be related to the lack of path delimiters in the second line of the error output:
```
Module not found: Error: Can't resolve 'C:ProjectsSomeProjectsrcPagesSomePageLayout.pug'
```

&nbsp;  
I did not have this issue before attempting to implement `complex-loader`. The loaders in the configuration example below work fine when used independently. I also had success bundling them together with `multi-loader` & `webpack-combine-loaders`, but they are very limited since they don't work with object-based loaders.

&nbsp;  
I tried debugging this @ `node_modules/**/complex-loader/*` by replacing path delimiters within `query` & `resourcePath` to both `/` & `\`, but those attempts didn't seem to affect anything. Don't really know enough about webpack internals to track it down beyond that.

&nbsp;  
### Loader Configuration

```javascript
{
  test: /\.pug$/,
  use: {
    loader: "complex-loader",
    options: {

      load_HTML: [
        {loader:"pug-loader", options:{pretty: true}},
      ],

      copy_Files: [
        {
          loader: "file-loader",
          options: {
            name: "[name].[ext]",
            outputPath: function(url, resourcePath, context)
              {return "./Some/Dynamic/Path/"}
          }
        }
      ],

    }
  }
```

**Comment by morulus** [comment]
Thanks for the issue. I apologize for the delayed response. I'll try to find time to reproduce an error and make a fix.

**Comment by morulus** [comment]
Resolved in [v0.2.0](https://github.com/morulus/complex-loader/releases/tag/v0.2.0)
